lib/account.php

```diff
@@ -92,6 +93,16 @@ public function getImapConnection() {
 			if ($this->config->getSystemValue('app.mail.imaplog.enabled', false)) {
 				$params['debug'] = $this->config->getSystemValue('datadirectory') . '/horde.log';
 			}
+			if ($this->config->getSystemValue('app.mail.server-side-cache.enabled', false)) {
+				$cacheFac = \OC::$server->getMemCacheFactory();
```

Comment:
for sure .. to be honest: I quickly hacked this together during a 2hrs management call :zzz:  :speak_no_evil: